modules/pruning-images-manual.adoc

```diff
@@ -198,6 +201,9 @@ You can apply conditions to your manually pruned images.
 *** Replica sets
 *** Build configurations
 *** Builds
+*** Jobs
+*** Cronjobs
+*** Stateful sets
 * There is no support for pruning from external registries.
```

Comment:
I would not have these 3 items be bullets because they aren't part of the conditions for pruning in the list (I think?). For example:

> There is no support for pruning from external registries.
> 
> When an image is pruned, all references to the image are removed from all image streams that have a reference to the image in status.tags. Image layers that are no longer referenced by any images are removed.

Reply:
They should be in the list as bullets as they are actually a part of the conditions for pruning which is exactly what I'm trying to change in the doc. They were added in the pruning conditions as part of https://bugzilla.redhat.com/show_bug.cgi?id=1880068 since OCP 4.7 and the doc hasn't been updated since.